README.md

```diff
@@ -4,10 +4,14 @@ No-fuss hapi server testing
 
 [![Build Status](https://travis-ci.org/devinivy/labbable.svg?branch=master)](https://travis-ci.org/devinivy/labbable) [![Coverage Status](https://coveralls.io/repos/devinivy/labbable/badge.svg?branch=master&service=github)](https://coveralls.io/github/devinivy/labbable?branch=master)
 
+Lead Maintainer — [Devin Ivy](https://github.com/devinivy)
+
+## Introduction
+
 It can be a pain to get your hapi server into your tests, especially when using otherwise wonderful tools such as **[glue](https://github.com/hapijs/glue)**.  Labbable makes this process very simple, and encourages the best practice of testing an initialized (but not started) hapi server.
 
 ##### Why initialize the server for tests?
-Plugin dependencies are only enforced at the time of [server initialization](https://github.com/hapijs/hapi/blob/master/API.md#serverinitializecallback).  This means code that relies on a plugin being present (typically by the `after` callback of [`server.dependency(deps, after)`](https://github.com/hapijs/hapi/blob/master/API.md#serverdependencydependencies-after)) will only run during initialization.  And if there are any dependencies missing, those errors will surface only during initialization.  Your server's caches will also be started and `onPreStart` server extensions will run.
+Plugin dependencies are only enforced at the time of [server initialization](https://hapijs.com/api#serverinitializecallback).  This means code that relies on a plugin being present (typically by the `after` callback of [`server.dependency(deps, after)`](https://hapijs.com/api#serverdependencydependencies-after) will only run during initialization.  And if there are any dependencies missing, those errors will surface only during initialization.  Your server's caches will also be started and `onPreStart` server extensions will run.
```

Comment:
Elsewhere we've started referencing the v16 branch for links rather than the hapi site– can we keep doing that?

Reply:
Dah! yes, thanks for catching that